mitodl/bootcamp-ecommerce
# Bootcamp index page and routing - #170

#### What are the relevant tickets?
(Required)

#### What's this PR do?
Adds a `BootcampIndexPage` to act as a router for all `BootcampRunPage` objects. This class has placeholder routing in place (currently only 404s when trying to serve the index page and slug-based default behaviour for bootcamp run pages) which we can later customise to implement our own courseware-ID based routing if required (like in xPRO).

#### How should this be manually tested?
Create a `BootcampIndexPage` under `HomePage`. Move any existing (or create new) `BootcampRunPage` under the index page. Click "view live" on the run page from inside the CMS, it should navigate to `/bootcamps/{run_page_slug}/` and the page should render as expected.
